packages/common/src/index.ts

```diff
@@ -1389,3 +1390,24 @@ export const convertAdditionalMetric = ({
         metric: additionalMetric,
         tableLabel: table.label,
     });
+
+export const getTableMagicMetrics = (
```

Comment:
Why are we adding this to common instead of just frontend. Are you planning to use this on the backend at some point ?